src/Tribe/Event_Repository.php

```diff
@@ -57,18 +106,18 @@ public function filter_by_cost( $value, $operator = '=', $symbol = null ) {
 			'NOT BETWEEN',
 			'IN',
 			'NOT IN',
-		) ) ) {
+		] ) ) {
 			throw Tribe__Repository__Usage_Error::because_this_comparison_operator_is_not_supported( $operator, 'filter_by_cost' );
 		}
 
-		if ( in_array( $operator, array(
+		if ( in_array( $operator, [
 				'BETWEEN',
 				'NOT BETWEEN',
-			) ) && ! ( is_array( $value ) && 2 === count( $value ) ) ) {
+			] ) && ! ( is_array( $value ) && 2 === count( $value ) ) ) {
```

Comment:
* :no_entry_sign: Multi-line function call not indented correctly; expected 8 spaces but found 12
* :no_entry_sign: Array closer not aligned correctly; expected 8 space(s) but found 12
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 7c0c9553bd16d8ca1aaa356a2dbd0cc8</sub>